sql/core/src/test/resources/sql-tests/results/predicate-functions.sql.out

```diff
@@ -1,5 +1,5 @@
 -- Automatically generated by SQLQueryTestSuite
--- Number of queries: 31
+-- Number of queries: 32
```

Comment:
It sounds like the last run does not use our SPARK_GENERATE_GOLDEN_FILES to generate the result file.